optuna/integration/skopt.py

```diff
@@ -81,8 +82,8 @@ def objective(trial):
         consider_pruned_trials:
             If this is :obj:`True`, the PRUNED trials are considered for sampling.
 
-            .. note::
-                Added in v2.0.0 as an experimental feature. The interface may change in newer
+            .. versionadded:: 2.0.0
```

Comment:
This change seem somewhat orthogonal to the intention but could you comment on the motivation? I personally find `.. note::` to be easier to notice in the docs.

Reply:
I just followed following Optuna's style convention. But as you said, it seemed orthogonal with the template of experimental decorator.
https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#write-versionadded-directive-to-the-doc-of-new-features

I'm not sure which directives should be used in Optuna, but I reverted this change for now.
Thanks!

Reply:
Me neither. I haven't exactly understood when to use `versionadded`. It might be suitable for experimental features since the version number is a part of it, but at the same time, it seems like a very general features given the brief documentation by sphinx. 🤔